sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeReceiptsOperation.cs

```diff
@@ -2,31 +2,71 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
 using Azure.Core.Pipeline;
-using System.Linq;
-using System.Collections.Generic;
 
 namespace Azure.AI.FormRecognizer.Models
 {
-    internal class ExtractReceiptOperation : Operation<IReadOnlyList<ExtractedReceipt>>
+    /// <summary>
+    /// Tracks the status of a long-running operation for recognizing values from receipts.
+    /// </summary>
+    public class RecognizeReceiptsOperation : Operation<IReadOnlyList<ExtractedReceipt>>
     {
+        /// <summary>Provides communication with the Form Recognizer Azure Cognitive Service through its REST API.</summary>
+        private readonly ServiceClient _serviceClient;
+
+        /// <summary>The last HTTP response received from the server. <c>null</c> until the first response is received.</summary>
         private Response _response;
+
+        /// <summary>The result of the long-running operation. <c>null</c> until result is received on status update.</summary>
         private IReadOnlyList<ExtractedReceipt> _value;
-        private bool _hasCompleted;
 
-        private readonly ServiceClient _operations;
+        /// <summary><c>true</c> if the long-running operation has completed. Otherwise, <c>false</c>.</summary>
+        private bool _hasCompleted;
 
+        /// <inheritdoc/>
         public override string Id { get; }
 
+        /// <inheritdoc/>
         public override IReadOnlyList<ExtractedReceipt> Value => OperationHelpers.GetValue(ref _value);
 
+        /// <inheritdoc/>
         public override bool HasCompleted => _hasCompleted;
 
+        /// <inheritdoc/>
         public override bool HasValue => _value != null;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RecognizeReceiptsOperation"/> class.
+        /// </summary>
+        /// <param name="operationId">The ID of this operation.</param>
+        /// <param name="client">The client used to check for completion.</param>
+        public RecognizeReceiptsOperation(string operationId, FormRecognizerClient client)
```

Comment:
if it applies, same comment here about a common constructor

Reply:
Replied in the other conversation.